pkg/common/scheduler/placement.go

```diff
@@ -38,3 +38,18 @@ func GetPlacementForPod(placements []duckv1alpha1.Placement, podName string) *du
 	}
 	return nil
 }
+
+// GetPodCount returns the number of pods with the given placements
+func GetPodCount(placements []duckv1alpha1.Placement) int {
+	set := map[string]int{}
+	for _, p := range placements {
+		if p.VReplicas > 0 {
+			if _, ok := set[p.PodName]; !ok {
+				set[p.PodName] = 1
+				continue
+			}
+			set[p.PodName] += 1
```

Comment:
Since we don't really use the map value here, no need to increment the count. Consider using a set instead of a map map[string]interface{}

Reply:
Actually I can improve this after we merge this PR.

Reply:
I did it. Thx